Packs/EclecticIQ/Integrations/EclecticIQV2/EclecticIQV2.yml

```
@@ -0,0 +1,174 @@
+      description: EclecticIQ Entity Title
+      type: string
+    description: Lookup observables from EclecticIQ Intelligence Center Platform
+  - name: create_sighting
+    arguments:
+    - name: value
+      required: true
+      description: ' value for the sighting'
+      type: textArea
+    - name: description
+      required: true
+      description: description about the sighting
+      type: textArea
+    - name: title
+      required: true
+      description: ' Title for the sighting'
+      type: textArea
+    - name: tags
+      required: true
+      default: true
+      description: ' Tag for the sighting'
+      type: textArea
+    - name: type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - ipv4
+      - ipv6
+      - domain
+      - uri
+      - email
+      - hash-md5
+      - hash-sha256
+      - hash-sha1
+      - hash-sha512
+      description: type for the sighting
+      type: textArea
+    - name: confidence_level
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - low
+      - medium
+      - high
+      - unknown
+      description: severity level of the sighting
+      type: textArea
+    outputs:
+    - contextPath: Sighting.Data.Type
+      description: Sighting Type
+      type: string
+    - contextPath: Sighting.Data.tags
+      description: Sighting Tags
+      type: string
+    - contextPath: Sighting.Data.title
+      description: Sighting Title
+      type: string
+    - contextPath: Sighting.Data.description
+      description: Sighting Description
+      type: string
+    - contextPath: Sighting.Data.value
+      description: Sighting Value
+      type: string
+    description: 'create sighting in the EclecticIQ Intelligence Center Platform '
+  - name: create_observable
+    arguments:
+    - name: type
+      required: true
+      description: Type of the observable
+      auto: PREDEFINED
+      predefined:
+      - ipv4
+      - ipv6
+      - domain
+      - uri
+      - email
+      - hash-md5
+      - hash-sha1
+      - hash-sha256
+      - hash-sha512
+      type: textArea
+    - name: value
+      required: true
+      default: true
+      description: value of the type of observable
+    - name: maliciousness
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - unknown
+      - safe
+      - low
+      - medium
+      - high
+      description: severity level of the type
+    outputs:
+    - contextPath: Observables.Data.type
+      description: Observable Type
+      type: string
+    - contextPath: Observables.Data.value
+      description: Observable Value
+      type: string
+    - contextPath: Observables.Data.maliciousness
+      description: Observable maliciousness
+      type: string
+    description: 'create observable in the EclecticIQ Intelligence Center Platform '
+  runonce: true
+  dockerimage: demisto/python3:3.10.9.40422
+tests:
+- No test - instance issues
+fromversion: 5.0.0
```

```suggestion
fromversion: 6.5.0
```
Since it is a new content, as newer the version- you can use newer XSOAR features in the integration, without client's needing support in older versions.

---

Updated